definition_tooling/converter/converter.py

```diff
@@ -80,35 +81,36 @@ def _parse(cls, version):
         return cls.parse(version)
 
     @classmethod
-    def __get_validators__(cls):
+    def __get_pydantic_core_schema__(
+        cls, source_type, _handler
+    ) -> core_schema.CoreSchema:
         """Return a list of validator methods for pydantic models."""
-        yield cls._parse
+        return core_schema.no_info_wrap_validator_function(
+            cls.parse,
+            core_schema.str_schema(),
+            serialization=core_schema.to_string_ser_schema(),
+        )
 
     @classmethod
-    def __modify_schema__(cls, field_schema):
+    def __get_pydantic_json_schema__(cls, _core_schema, handler) -> JsonSchemaValue:
         """Inject/mutate the pydantic field schema in-place."""
-        field_schema.update(
-            examples=[
-                "1.0.2",
-                "2.15.3-alpha",
-                "21.3.15-beta+12345",
-            ]
-        )
+        return handler(core_schema.str_schema())
 
 
 class DataProductDefinition(BaseModel):
     version: PydanticVersion = "0.0.1"
     deprecated: bool = False
     description: str
     error_responses: Dict[ERROR_CODE, ErrorModel] = {}
-    name: Optional[str]
+    name: str = ""
```

Comment:
Is this field supposed to be optional? What do we do with an empty string as the name? Where is this value used in general?